src/components/Header/Header.jsx

```diff
@@ -0,0 +1,33 @@
+import { Link } from 'react-router-dom'
+import mainLogo from "../../assets/mainLogo.png"
+import "./Header.css"
+
+export default function Header() {
+  return (
+    <header className="header shadow">
+        <Link to="/">
```

Comment:
You can add NavLink here for navigation.

Reply:
I'll check that out, thank you!